codalab/apps/web/views.py

```diff
@@ -1580,6 +1583,19 @@ def download_dataset(request, dataset_key):
         return HttpResponse(msg, status=400, content_type='text/plain')
 
 
+def publish_dataset(request, dataset_key):
```

Comment:
maybe "toggle_dataset_publish" or something like that? since it will also _un_ publish?

Reply:
✅ yeah i was thinking the same